app/views.py

```diff
@@ -87,8 +87,19 @@ def login():
             return "There was an error.", 500
         # Else success, redirect to index page
         else:
-            session['username'] = username
             session['token'] = token['token']
+            user = getUser(username)
+
+            if type(user) is list:
```

Comment:
@aaroncohen73 I get that. Which is why this check should not exist. The function we are calling to set the `user` variable should just return the user and not a list containing the user.
What we have now:
```python
user = getUser(username)

if type(user) is list:    # A silly check to make because that function shouldn't return a list data type.
    user = user[0]
```

What we should have
```python
user = getUser(username)    #only returns user object and not list because why would it?
```